crc/services/study_service.py

```diff
@@ -105,8 +106,21 @@ def get_study(study_id, categories: List[WorkflowSpecCategory], study_model: Stu
                     study.warnings = StudyService._update_status_of_workflow_meta(workflow_metas,
                                                                                   master_workflow_results)
                 category.workflows = workflow_metas
+        # Calculate study progress and return it as a integer out of a hundred
```

Comment:
Are we calcuating this on the fly every time (and then it doesn't need to be in the database) OR
are we storing it in the Database (and it doesn't need to be calculated everytime).  
I'm kind of for the former, since its "Progress" and likely to change.  But not digging the idea of doing both things all the time.